tests/lib/tools/nested-state

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: prepare"
+    echo "       restore"
+    echo "       build-core-image"
+    echo "       start-new-core-vm [--param-cd PARAM] [--param-mem PARAM]"
```

Comment:
What does `cd` stand for? Is it `cdrom`?

Reply:
yes, changed